### DISCUSSION TITLE
Is it possible to do a qualified association

### DISCUSSION THREAD

**Discussion by PaulMaget** [gaphor/gaphor#2680] [Q&A]
Gaphor is by far the best software I've seen to create class diagrams. The only thing I can't find how to do is qualified associations.
I'm using it for school so I don't want to get my team stuck on a project because of it.

Is there a way to do qualified associations in Gaphor?
If not, is there a workaround? Is it planned for a later update?

**Comment by amolenaar** [comment]
Hi @PaulMaget,

Sorry for the late response.

We do not have support for qualified associations at the moment. Apart from association ends names, we do not have an alternative at the moment. If you think it's a feature Gaphor really needs, please open a Issue.
